src/api/FileVersionAPI.js

```diff
@@ -77,11 +77,20 @@ class FileVersionAPI extends API {
      * @param {Params} queryParams - Key-value map of querystring params
      * @return {void}
      */
-    successHandler = (data: Data, queryParams: Params): void => {
+    successHandler = (data: Data, queryParams: Params): Promise<any> => {
+        if (data.type === 'error' || !Array.isArray(data.entries)) {
+            const error = new Error(`Could not read annotations from file version with ID ${this.fileVersionId}`);
+            this.emit(ANNOTATOR_EVENT.error, {
```

Comment:
Does the API need to emit events, or can the just parent .catch handle emitting?

Reply:
Also, can this be separated into an error handler?